platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -184,6 +184,8 @@ event_processor:
 #                  is the same as the `signing_key`, the callback will be activated.
 #   - callback_url: (optional) the URL of the client's callback API endpoint. Optional due to some wallets may opt to
 #                  poll instead, or may use polling first before implementing callbacks at a later stage.
+#   - destination_accounts: (optional) list of accounts allowed to be used for the deposit. By default, only SEP-10
+#                  authenticated account can be used to deposit funds into.
```

Comment:
I think we're missing an entry for `allow_any_destination` here right? Its in the example below